geometry_msgs/PoseStamped[] grasping_poses